PureJava/org.eclipse.nebula.visualization.xygraph/src/org/eclipse/nebula/visualization/xygraph/figures/XYGraph.java

```diff
@@ -593,13 +688,9 @@ public OperationsManager getOperationsManager() {
 	public void performAutoScale() {
 		final ZoomCommand command = new ZoomCommand("Auto Scale", xAxisList, yAxisList);
 		for (Axis axis : xAxisList) {
-			if (!axis.isVisible())
```

Comment:
It is not present in upstream, but so far it doesn't seem to break anything. Have you tested?